api/handler.go

```diff
@@ -186,6 +190,20 @@ func (h *apiHandler) usageHandler() chi.Router {
 	return router
 }
 
+func (h *apiHandler) aiHandler() chi.Router {
+	router := chi.NewRouter()
+	if h.usage == nil {
```

Comment:
Shouldn't this be `h.ai`?
```suggestion
	if h.ai == nil {
```